K8s-deployment/Charts/data-ingestion/example-secrets/secrets/config.json

```diff
@@ -29,6 +37,9 @@
 			"catServerHost": "<catalogue-api-public-domain>",
 			"catServerPort": 443,
 			"jwtIgnoreExpiry": false
+			"audience": "rs.iudx.io",
```

Comment:
audience field already defined above as ``			"audience": "<rs-audience-domain>",``

Following options are not required, please remove. Will notify di server repo also
 "resourceServerId": "",
 "serverMode": ""

Reply:
https://github.com/datakaveri/iudx-data-ingestion-server/pull/71